File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.java
class Solution {
    public String convertToTitle(int columnNumber) {
        String result = "";
        while(columnNumber > 0){
            columnNumber--;
            int rem = columnNumber % 26;
            result = (char) (rem + 'A') + result;
            columnNumber = columnNumber / 26;
        }
        return result;
    }
}

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        result = ""
        while columnNumber > 0:
            columnNumber -= 1
            rem = columnNumber % 26
            result = chr(rem + ord('A')) + result
            columnNumber = columnNumber // 26
        return result
        